northstar/src/npk/npk.rs

```diff
@@ -735,6 +735,8 @@ fn gen_pseudo_files(manifest: &Manifest) -> Result<NamedTempFile, Error> {
                 }
             }
         })
+        // This directory is used by `pivot_root` as the `old-root` parameter
+        .chain(pseudo_directory(Path::new("/mnt"), 755))
```

Comment:
What about naming to `old-root`?

Reply:
Outdated. The additional `met` is not needed if both `pivot_root` arguments are the new root.